src/GoogleAuthenticator.php

```diff
@@ -86,6 +86,21 @@ public function checkCode($secret, $code): bool
         return $result > 0;
     }
 
+    final protected function setPeriod($period):bool
```

Comment:
For consistency I am for setCodePeriod()

Reply:
Yes, I would agree with that.

Reply:
a subclass can use it, but can't overwrite it, no one outside can use it. That is the point, right?

Reply:
It was supposed to be, but the class itself is final (see the issue update I did).

Reply:
If the class is final it can be final public but also only public, or an i missing sth? 🤔

Reply:
The original request was for inherited classes.  If there is no issue making the codePeriod public, I can switch that easily.

Reply:
Yes public function is perfect in this case